grpc.go

```diff
@@ -524,6 +525,76 @@ func (a *agentGRPC) rollbackFailingContainerCreation(ctr *container) {
 	}
 }
 
+func changeToBundlePath(spec *specs.Spec) (string, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return cwd, err
+	}
+
+	rootfsPath := spec.Root.Path
+	if !filepath.IsAbs(rootfsPath) {
+		rootfsPath = filepath.Join(cwd, rootfsPath)
+	}
+
+	bundlePath := filepath.Dir(rootfsPath)
+
+	err = os.Chdir(bundlePath)
+	if err != nil {
+		return cwd, err
+	}
+
+	return cwd, nil
+}
+
+func writeSpecToFile(spec *specs.Spec) error {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	configPath := path.Join(cwd, "config.json")
```

Comment:
Once this function is in `protocols/grpc/utils.go`, you could move `ociConfigFile` from `protocols/grpc/utils_test.go` into `protocols/grpc/utils.go` to avoid hard-coding it.